### PR TITLE
CA-247342: Add auto-complete for sdn-controller CLI commands.

### PR DESCRIPTION
**Pull request by huizh**
Signed-off-by: Hui Zhang <hui.zhang@citrix.com>

### CODE REVIEW

File: ocaml/xe-cli/bash-completion [hunk 1]
```diff
@@ -108,6 +108,9 @@ _xe()
                     host-cpu-*|host-crashdump-*|gpu-group-*|vgpu-type-*|pvs-server-*|pvs-proxy-*|pvs-site-*)
                         # Chop off at the second '-' and append 'list'
                         cmd="$(echo ${OLDSTYLE_WORDS[1]} | cut -d- -f1-2)-list";;
+                    sdn-controller-*)
```
Comment (robhoes) [inline ocaml/xe-cli/bash-completion]: Let's just add a case to the one above instead.

Comment (huizh) [inline ocaml/xe-cli/bash-completion]: Updated. Thanks.

Comment (robhoes) [inline ocaml/xe-cli/bash-completion]: Sorry, that's not what I meant. I meant this:

    host-cpu-*|host-crashdump-*|gpu-group-*|vgpu-type-*|pvs-server-*|pvs-proxy-*|pvs-site-*|sdn-controller*)

Comment (huizh) [inline ocaml/xe-cli/bash-completion]: Sorry I misunderstood. Now updated. :)

**Pull request merged**